### PR TITLE
Update wedding details km

### LINKED ISSUE #58: Bugs in test files
- [ ] Fix Vendor spelling on VendorDashboard.css
- [ ] Fix `Guest` test to reflect new button (X instead of trashcan/delete`

### LINKED ISSUE #59: Bug from vendordashboard merge

### LINKED ISSUE #60: User should be able to navigate home by clicking on logo

### PR DESCRIPTION
**Pull request by Kristiannmiller**
# Pull Request

## What's this PR do? 

- [X] New feature (non-breaking change which adds functionality)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Testing
- [X] Styling
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Description of changes?
- Adds new styled component BackButton
- Adds additional styling to GuestList component
- Adds additional styling to WeddingDetails component
- Fixes merge bugs in VendorDashboard
- Fixes spelling error in VendorDashboard and throughout app
- Adds navigation through main logo back to VendorDashboard

## What are the relevant tickets? 
* Closes #58 
* Closes #59 
* Closes #60

### CODE REVIEW

File: src/components/App/App.tsx [hunk 1]
```diff
@@ -15,7 +15,9 @@ const App = () => {
   return (
     <div className="appWrap">
       <header className="headerWrap">
-        <img src={logo} className="logo" alt="Matrimania Logo"/>
+        <Link to={`/vendor-dashboard`}>
+          <img src={logo} className="logo" alt="Matrimania Logo"/>
+        </Link>
```
Comment (ericberglund117) [inline src/components/App/App.tsx:20]: I like the ability to navigate back to this page via the logo

File: src/components/App/styledComponents.styles.ts [hunk 2]
```diff
@@ -121,3 +122,54 @@ export const StyledButton = styled.div`
...
+    left: -200px;
+    position: absolute;
+    padding: 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    transition: all .35s ease-Out;
+    bottom: 0;
+  }
+`
```
Comment (ericberglund117) [inline src/components/App/styledComponents.styles.ts:175]: The back and done button look fantastic with the arrow animation

**Review by ericberglund117 — APPROVED** [review]
All this looks amazing!

File: src/components/App/styledComponents.styles.ts (hunk 2 above)
Comment (hannakim91) [reply src/components/App/styledComponents.styles.ts:175]: Makes the page look a lot more complete!

File: src/components/VendorDashboard/VendorDashboard.tsx [hunk 3]
```diff
@@ -1,12 +1,32 @@
...
 import WeddingCard from '../WeddingCard/WeddingCard'
 import { getWeddings } from '../../apiCalls'
+import { Link } from 'react-router-dom';
 import { StyledButton } from '../App/styledComponents.styles'
-import './VenderDashboard.css';
+import './VendorDashboard.css';
 
+type Wedding = {
+  id: number;
+  name: string;
```
Comment (hannakim91) [inline src/components/VendorDashboard/VendorDashboard.tsx:11]: Thanks for fixing these merge conflicts!!

**Review by hannakim91 — APPROVED** [review]

**Pull request merged**